src/confdb/confdb.c

```diff
@@ -43,6 +48,34 @@
 #define RETRIEVE_DOMAIN_ERROR_MSG "Error (%d [%s]) retrieving domain [%s], "\
                                   "skipping!\n"
 
+#ifdef LDB_MODULES_PATH_OVERRIDE
+static errno_t set_ldb_modules_path(void)
+{
+    static bool initialized = false;
```

Comment:
Hi,

I would like to suggest to remove the number of `#ifdef`s by including `stdlib-h` and` errno.h` unconditionally and always  calling `set_ldb_modules_path()` and use a single `#ifdef` in `set_ldb_modules_path()` to either run the code or just return `EOK`. 

bye,
Sumit

Reply:
Hi,

thanks for the updates, what about reducing the `#ifdef`s?

bye,
Sumit